card.go

```diff
@@ -7,6 +7,15 @@ import (
 	"github.com/stripe/stripe-go/form"
 )
 
+// CardAvailablePayoutMethod is a set of available payout methods for the card.
+type CardAvailablePayoutMethod string
+
+// List of values that CardAvailablePayoutMethod can take.
+const (
+	CardAvailablePayoutMethodStandard CardAvailablePayoutMethod = "Standard"
+	CardAvailablePayoutMethodInstant  CardAvailablePayoutMethod = "Instant"
```

Comment:
Sorry for the nit, but mind ordering this list alphabetically? (We've made an effort to do that ~everywhere throughout the SDK.)

ptal @jack-stripe